README.md

```diff
@@ -96,6 +96,41 @@ class Point:
   Schema: ClassVar[Type[Schema]] = Schema
 ```
 
+It is also possible to derive all schemas from your own base class
+```python
```

Comment:
It would be nice if we could make this example smaller. (see my suggestions below)